agents-for-bedrock/features-examples/09-fine-grained-access-permissions/003_bedrock-agent/agent/agent.py

```diff
@@ -0,0 +1,287 @@
+import json
+import boto3
+import os
+import logging
+import cognitojwt
+
+
+# Set up logging
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+verified_permissions_client = boto3.client("verifiedpermissions")
+
+
+def get_named_parameter(event, name):
+    return next(item for item in event['parameters'] if item['name'] == name)['value']
+
+
+def list_claims(event):
+    # TODO: Implement logic to retrieve and return a list of claims
+    dummy_claims = [
+        {
+            "id": 1,
+            "claimAmount": 1000.0,
+            "claimDescription": "Dummy claim 1",
+            "claimStatus": "approved"
+        },
+        {
+            "id": 2,
+            "claimAmount": 2500.75,
+            "claimDescription": "Dummy claim 2",
+            "claimStatus": "pending"
+        }
+    ]
+    return dummy_claims
+
+
+def get_claim(event):
+    claim_id = int(get_named_parameter(event, 'claimId'))
```

Comment:
get claim is not connected with the claims in list claims? would make sense to have the same claim on the list claims

Reply:
Updated the list so that the dummy claims are matching